keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/artifacts/Build.kt

```diff
@@ -0,0 +1,25 @@
+package com.netflix.spinnaker.keel.api.artifacts
+
+/**
+ * An immutable data class that represents a published software artifact in the Spinnaker ecosystem.
+ *
+ * This class mirrors [com.netflix.spinnaker.igor.build.model.GenericBuild], but without all the Jackson baggage.
+ */
+
+data class Build(
+  val building: Boolean = false,
+  val fullDisplayName: String? = null,
+  val name: String? = null,
+  val number: Int = 0,
+  val duration: Int? = null,
+  /** String representation of time in nanoseconds since Unix epoch  */
+   val timestamp: String? = null,
+
+  val result: Any? = null,
+  val artifacts: List<Any>? = null,
+  val testResults: List<Any>? = null,
```

Comment:
Curious about the use of `Any` here. Do we not know the structure of these objects?

Reply:
we don't. This object is a copy from Igor `GenericBuild` object. My goal is to have this object to suit different CI providers, so I don't want to enforce any kind of structure here.